### DISCUSSION TITLE
Genymotion GPS feature not working on M2 MacBook pro

### DISCUSSION THREAD

**Discussion by notkojak** [Genymobile/feedback#71] [Genymotion Desktop] [answered]
I'm trying to use genymotion to run some automation on an app that requires location.

When I use my M2 MacBook, android 11 is the only version available. The emulator works fine. I'm able to install OpenGApps, and I'm able to install the app I need from the play store. But when using the GPS spoofing feature in genymotion, it doesn't work in any app. I tried it in google maps, and when I click "current location" it seems to hang endlessly. I've tried using 3rd party GPS spoofing apps from the play store, but those will not work either because they all seem to require my current location as well. Some of these apps give the message "google play services are updating". I tried installing Google Play Services, and updating them, but it did not help. I tried turning off and on wifi and bluetooth for location as well. No luck.

Out of curiosity I tried this again on my old intel, using android 6, 7, and 9. There, the GPS feature works perfectly, but I'm unable to install the app I need from the play store and get "this app is not available for your device", but I cannot figure out which criteria the VM is not meeting.

**Comment by guillaume-roche** [comment, marked as answer]
Hello,

We are aware of this issue happening on Macs M1/M2. We will ship a new system image to fix it.

Best regards,
Guillaume
